starport/services/networkbuilder/networkbuilder.go

```diff
@@ -39,6 +39,7 @@ const (
 
 var (
 	sourcePath = filepath.Join(services.StarportConfDir, "spn-chains")
+	homePath   = filepath.Join(os.ExpandEnv("$HOME"), ".spn-chain-homes")
```

Comment:
https://golang.org/pkg/os/#UserHomeDir

Reply:
We have used `$HOME` until then. I would say we should replace all of the occurrences in a specific PR

Reply:
https://github.com/tendermint/starport/issues/974

Reply:
okay.